docs/admin/runai-setup/authentication/sso.md

```diff
@@ -106,9 +83,10 @@ You can configure your IdP to map several IdP attributes:
         1. In the `Discovery URL` field, enter the discovery URL .
         2. In the `Client ID` field, enter the client ID.
         3. In the `Client Secret` field, enter the client secret.
-	4. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional).  Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
-        5. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
-        6.When complete, press `Save`.
+		4. Copy the `Redirect URL` and use it in your identity provider.
+		5. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional).  Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
+        6. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
+        7.When complete, press `Save`.
```

Comment:
```suggestion
	4. Copy the `Redirect URL` and use it in your identity provider.
	5. Add the OIDC scope to be used during authentication to authorize access to a user's details (optional).  Each scope returns a set of user attributes.  The scope must match the names in your identity provider.
        6. In the `User attributes` field enter the attribute and the value in the identity provider. (optional)
        7.When complete, press `Save`.
```
@ozRunAI Please be careful of the spaces you add. If these are meant to be indented ordered list, then you need to add spaces in the lines above before you indent them.